src/daft-core/src/array/ops/utf8.rs

```diff
@@ -1328,6 +1343,44 @@ impl Utf8Array {
         Ok(result)
     }
 
+    pub fn normalize(&self, opts: Utf8NormalizeOptions) -> DaftResult<Utf8Array> {
+        let whitespace_regex = regex::Regex::new(r"\s+").expect("whitespace regex should compile");
```

Comment:
super nit: you can probably just `.unwrap()` here since it's either correctly written or not